Library/Homebrew/env_config.rb

```diff
@@ -477,7 +477,7 @@ module EnvConfig
         description: "A comma-separated list of hostnames and domain names excluded " \
                      "from proxying by `curl`(1), `git`(1) and `svn`(1) when downloading through Homebrew.",
       },
-    }.freeze
+    }.freeze, T::Hash[Symbol, T.untyped])
```

Comment:
Can probably make this slightly stronger to cover at least the top level of nesting:

```suggestion
    }.freeze, T::Hash[Symbol, T::Hash[Symbol, T.untyped]])
```